integrations/slack/webhooks/bot_events.go

```diff
@@ -21,6 +21,7 @@ var BotEventHandlers = map[string]BotEventHandler{
 	"app_mention": events.AppMentionHandler,
 	// TODO: app_rate_limit
 	// TODO: app_uninstalled
+	"app_home_opened": events.AppHomeOpenHandler,
```

Comment:
Nit: let's keep this ordered alphabetically, i.e. put this line above "app_mention"